<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="runner.TestRunner" tests="1" time="35.126" ignored="0" failures="0" timestamp="20 Jun 2018 19:36:07 GMT" skipped="0" hostname="Thangaraj" errors="1">
  <testcase name="feature" time="35.126" classname="runner.TestRunner">
    <error type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: No element found with identifier name-username">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: No element found with identifier name-username
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at runner.TestRunner.feature(TestRunner.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: No element found with identifier name-username
	at org.junit.Assert.fail(Assert.java:88)
	at actions.CommonActions.clearTextEvent(CommonActions.java:109)
	at actions.WebActions.actionClearText(WebActions.java:134)
	at pagefactory.LoginPage.actionClearInputsInLoginPage(LoginPage.java:66)
	at scriptrunner.ScriptRunnerOne.run(ScriptRunnerOne.java:37)
	at âœ½.And login with multiple Username and Password(EspressoMultiLogin.feature:6)
]]>
    </error>
  </testcase> <!-- feature -->
</testsuite> <!-- runner.TestRunner -->
